user_guide_src/source/installation/upgrade_450.rst

```diff
@@ -125,6 +125,11 @@ Removed Deprecated Items
 Some deprecated items have been removed. If you extend these classes and are
 using them, upgrade your code. See :ref:`ChangeLog <v450-removed-deprecated-items>` for details.
 
+Auto Routing (Legacy) 
+=======================
+
+If the auto routing cannot found the controller, it will throw page not found exception before the filter executed.
+
```

Comment:
It is better that the place is before "Removed Deprecated Items", and
what to do should be written.

```diff
--- a/user_guide_src/source/installation/upgrade_450.rst
+++ b/user_guide_src/source/installation/upgrade_450.rst
@@ -119,17 +119,24 @@ has been removed.
 
 If you extneds ``BaseModel``, implement the ``getIdValue()`` method in the child class.
 
+Auto Routing (Legacy)
+=====================
+
+In previous versions, the controller filters might be executed even when the
+corresponding controller was not found.
+
+This bug has been fixed and now a ``PageNotFoundException`` will be thrown and
+the filters will not be executed if the controller is not found.
+
+If you have code that depends on this bug, for example if you expect global filters
+to be executed even for non-existent pages, please add the necessary routes.
+
 Removed Deprecated Items
 ========================
 
 Some deprecated items have been removed. If you extend these classes and are
 using them, upgrade your code. See :ref:`ChangeLog <v450-removed-deprecated-items>` for details.
 
-Auto Routing (Legacy) 
-=======================
-
-If the auto routing cannot found the controller, it will throw page not found exception before the filter executed.
-
 Breaking Enhancements
 *********************
 
```

Reply:
Done.